annotation-processing/src/main/java/com/exactpro/epfast/annotation/processing/ClassDescription.java

```diff
@@ -0,0 +1,29 @@
+package com.exactpro.epfast.annotation.processing;
+
+public class ClassDescription {
+    private String packageName;
+
+    private String className;
+
+    public ClassDescription(String packageName, String className) {
```

Comment:
initialize it with full type name. Then you have two options: split it to package/name in constructor or extract the corresponding parts in getters